src/main/java/Bundle_en.properties

```diff
@@ -419,13 +420,14 @@ dashboard.license.noActiveLicensesWarning=There must be at least one active lice
 dashboard.license.displayOrder=Display order
 dashboard.license.universalLicenseName=Universal license name
 dashboard.license.licenseURL=License URL
-dashboard.license.icon=Icon
+dashboard.license.badge=Badge
 dashboard.license.activeByDefault=Active by default
 dashboard.license.newLicense.add=Add new license
 dashboard.license.newLicense.universalName=Universal name
-dashboard.license.newLicense.chooseLicenseBadge=Select license badge
-dashboard.license.newLicense.missingTextField=This field must not be empty.
+dashboard.license.newLicense.licenseBadge=Select license badge
```

Comment:
'License badge' or simply 'Badge' ?

Reply:
Maybe BadgeBtn would be more suitable

Reply:
Well, this is for text that names what have to be provided by the user in this place. It has nothing to do with button

Reply:
Ah you are right it's for the label, so maybe BadgeLabel? .Badge can mean many things depending on context.

Reply:
Just to be clear (i'm not sure if we thinking about the same thing). I'm writing above about property value, not the key.

Reply:
ohhh I will ask Wojtek then since it's in the docs.